Mathlib/Topology/Algebra/SeparationQuotient/Basic.lean

```diff
@@ -195,6 +212,20 @@ instance instGroup [Group G] [TopologicalGroup G] : Group (SeparationQuotient G)
 instance instCommGroup [CommGroup G] [TopologicalGroup G] : CommGroup (SeparationQuotient G) :=
   surjective_mk.commGroup mk mk_one mk_mul mk_inv mk_div mk_pow mk_zpow
 
+/-- Neighborhoods in the quotient are precisely the map of neighborhoods in the prequotient. -/
+theorem nhds_eq (x : G) :
+    nhds (mk x) = Filter.map mk (nhds x) :=
+  le_antisymm ((SeparationQuotient.isOpenMap_mk).nhds_le x) continuous_quot_mk.continuousAt
+
+/-- `SeparationQuotient.mk` as a `MonoidHom`. -/
+@[to_additive (attr := simps) "`SeparationQuotient.mk` as an `AddMonoidHom`."]
+def mkGroupHom [CommGroup G] [TopologicalGroup G]  : G →* SeparationQuotient G where
```

Comment:
Does this work with `Monoid G`?

Reply:
there is already `mkMonoidHom`. In this case is `mkGroupHom` just unnecessary?

Reply:
Yes, in that case this definition is a duplicate; you can verify this by writing
```lean
def mkGroupHom [CommGroup G] [TopologicalGroup G]  : G →* SeparationQuotient G := mkMonoidHom
```
(and then deleting it once you convince yourself that works)

Reply:
convinced and removed